launch/mavros_sitl.launch

```diff
@@ -1,7 +1,9 @@
 <launch>
-	<!-- vim: set ft=xml noet : -->
-	<!-- example launch script for PX4 based FCU's -->
+<!-- vim: set ft=xml noet : -->
+<!-- example launch script for PX4 based FCU's -->
 
+<arg name="ns"/>
```

Comment:
maybe for people who run mavros standalone for testing? added in b311f7302b508d1ed7fd3cfecd0bfbe47b324bde